FastLSTM.lua

```diff
@@ -3,22 +3,46 @@ local FastLSTM, parent = torch.class("nn.FastLSTM", "nn.LSTM")
 -- set this to true to have it use nngraph instead of nn
 -- setting this to true can make your next FastLSTM significantly faster
 FastLSTM.usenngraph = false
+FastLSTM.bn = false
 
-function FastLSTM:__init(inputSize, outputSize, rho)
-   parent.__init(self, inputSize, outputSize, rho, false)
+function FastLSTM:__init(inputSize, outputSize, rho, eps, momentum, affine)
+   --  initialize batch norm variance with 0.1
+   self.eps = self.eps or 0.1
+   self.momentum = self.momentum or 0.1 --gamma
+   self.affine = self.affine or true
+   self.bn = self.bn
+
+   parent.__init(self, inputSize, outputSize, rho, false, self.eps, self.momentum, self.affine) 
```

Comment:
you don't need to change line 15 as the last arguments are ignored by the parent.